### ISSUE TITLE
ci is failing

### ISSUE THREAD

**Issue by Zamiell** [hoperyy/sync-directory#56]
looks like ci has been failing for a long time, is that expected?

**Comment by hoperyy** [comment]
Yes. I've focused on `sync-directory` function stuff for a long time and skipped CI.

CI is really needed now and later.
